pkg/gatherers/clusterconfig/events_filtering.go

```diff
@@ -0,0 +1,76 @@
+package clusterconfig
+
+import (
+	"sort"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// filterEvents() returns events that occoured since last interval
+func getEventsForInterval(interval time.Duration, events *v1.EventList) v1.EventList {
+	oldestEventTime := time.Now().Add(-interval)
+	var filteredEvents v1.EventList
+	for i := range events.Items {
+		if isEventNew(&events.Items[i], oldestEventTime) {
+			filteredEvents.Items = append(filteredEvents.Items, events.Items[i])
+		}
+	}
+	return filteredEvents
+}
+
+// if LastTimestamp is zero then try to check the event series
+func isEventNew(event *v1.Event, oldestEventTime time.Time) bool {
```

Comment:
The comment here would deserve an update as well :) This comment is more suitable to the inner code.